src/olympia/amo/utils.py

```diff
@@ -953,19 +953,23 @@ def rm_stored_dir(self, dir_path):
             self.delete(dn)
 
 
-def attach_trans_dict(model, objs):
-    """Put all translations from all non-deferred translated fields from objs
-    into a translations dict on each instance."""
+def attach_trans_dict(model, objs, fields=None):
```

Comment:
nit: `field_names` is a more friendly interface - we could accept a list of names, so you don't have to do `addon._meta.get_field(name)` each time.